content/general/concepts/software-testing/software-testing.md

```diff
@@ -17,6 +17,12 @@ CatalogContent:
 
 **Software testing** is the process of verifying and evaluating the functionality of a software application or product. It reduces and eliminates bugs, and mitigates the long-term costs for a company in terms of any fixes and updates that would be needed.
 
+It is most common to initiate during the [design phase](https://www.codecademy.com/resources/docs/general/software-development-life-cyclee) and can even occur after the product is deployed into production. Different [testing methodologies](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) are available to suit different project objectives.
```

Comment:
```suggestion
It is most common to initiate software testing during the design phase but it can even take place after the deployment phase of the [software development life cycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle).
```